uxarray/helpers.py

```diff
@@ -556,3 +556,111 @@ def _normalize_in_place(node):
         raise RuntimeError("Input array should have a length of 3: [x, y, z]")
 
     return list(np.array(node) / np.linalg.norm(np.array(node), ord=2))
+
+
+@njit
+def _pt_within_gcr(pt_cart, gcr_end_pt_cart_1, gcr_end_pt_cart_2):
```

Comment:
Hi @rajeeja,

This is the function I mentioned that has problem with the nested array input
Initially, the function signature looks like this:
```python
def _pt_within_gcr(pt_cart, gcr_end_pts_cart):
```
Where ` gcr_end_pts_cart = [gcr_end_pt_cart_1, gcr_end_pt_cart_2]`, And in this case, numba will always throw an exception.